less/index.less

```diff
@@ -57,3 +58,158 @@ html, body {
     height: 100%;
 }
 
+// Font Variables
+@sfont: 1.0rem;
+@mfont: 1.2rem;
+@lfont: 1.6rem;
+
+// Color Variables
+@gray: @gray;
```

Comment:
This causes an error in LESS as you have given a variable a value of itself. Try setting `@gray` to equal some color value such as `gray`.